llvm/include/llvm/DebugInfo/DWARF/DWARFAcceleratorTable.h

```diff
@@ -793,6 +794,14 @@ class DWARFDebugNames : public DWARFAcceleratorTable {
       next();
       return I;
     }
+    reference operator[](size_type idx) {
+      return CurrentIndex->getNameTableEntry(idx + 1);
+    }
+
+    difference_type operator-(const NameIterator &other) const {
```

Comment:
Can we get some comments on context on the new operators? I don't see where they are used elsewhere in this patch.

Reply:
Updated.
They are used in the parallelForEach implementation.
parallelFor(0, End - Begin, [&](size_t I) { Fn(Begin[I]); });